README.md

```diff
@@ -14,66 +14,98 @@ Web and Cloud Functions are in this repo. When code is checked into master branc
 ## More details about the app
 You authenticate with Github, which triggers a [Cloud Function](https://cloud.google.com/functions/docs/) that grabs a lot of data from the [Github API](https://developer.github.com/v3/).  The initial data fills out your basic profile (e.g. [languages you code in](https://developer.github.com/v3/repos/#list-languages)) and creates a list of connections based on your pull request history or the other committers on projects that you have contributed to.
 
+## Setting up a dev environment
 
+### Install dev tools
 
-## Dev Setup
+Note: we're using yarn to manage npm dependencies, which is really helpful
+for making sure you are using the exact same set of modules that have been
+tested to work with this project. The following instructions specify some
+global installs and config, which shouldn't be required if you know what you
+are doing, but make it much easier to get started if you are new to some of
+this tech, and even for many of us who use it all the time!
 
-We've got our node app in `functions`, our web app in `webapp/`, and our ios app in `ios/`. Please navigate to their readme files for setup instructions.
-
-install
+These instructions use `yarn` and `nvm` which we recommend:
 * [yarn](https://yarnpkg.com) -- required for yarn.lock sanity
-* [nvm](https://github.com/creationix/nvm) -- recommended
+* [nvm](https://github.com/creationix/nvm) -- makes easier switching to and from
+  other node projects that you might have
 
 ```
 nvm use v6.9.1
+npm install -g firebase-tools
+npm install -g @angular/cli
 ng set --global packageManager=yarn
-cd webapp
+git clone https://github.com/all-the-hubbub/hubbub.git
+cd hubbub
+```
+
+### Set up project
+
+To run a dev instance of Hubbub, you will need your own firebase project to
+run the Cloud Functions.  Since we are calling an external service from a Cloud
+Function, we need to set up billing for the account.
+
+1. Go to the [Firebase console](https://console.firebase.google.com) and
+   create a new project called `hubbub-dev`
+2. Setup billing. The easiset way is to upgrade to the pay-as-you-go Blaze plan,
+   but you won't be charged unless your usage exceeds the [generous free tier](https://firebase.google.com/pricing/).
```

Comment:
@brendanlim I didn't see anything on the pricing page that says that if you upgrade to Blaze but use less than the free tier that you won't be billed.  I thought I remembered hearing that... and I'm sure it is documented somewhere that I probably missed.  In any case, can you confirm this statement about billing?